### PR TITLE
Scanpolicies: initial help

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
As per title :)

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: addOns/scanpolicies/src/main/javahelp/help/helpset.hs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE helpset
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp HelpSet Version 2.0//EN"
+         "http://java.sun.com/products/javahelp/helpset_2_0.dtd">
+<helpset version="2.0" xml:lang="en-GB">
+  <title>Scan Policies</title>
+
+  <maps>
+     <homeID>top</homeID>
```
Comment (thc202) [inline addOns/scanpolicies/src/main/javahelp/help/helpset.hs:9]: `scanpolicies`

File: addOns/scanpolicies/src/main/javahelp/help/map.jhm [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+    <mapID target="scanpolicies" url="contents/scanpolicies.html" />
```
Comment (thc202) [inline addOns/scanpolicies/src/main/javahelp/help/map.jhm:7]: I'd add `addon.` prefix to the target keys, makes it less likely to clash with core keys.

File: addOns/scanpolicies/src/main/javahelp/help/map.jhm [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    <mapID target="policy-default" url="contents/policy-default.html" />
```
Comment (thc202) [inline addOns/scanpolicies/src/main/javahelp/help/map.jhm:8]: And this one prefix `addon.scanpolicies.`.

**Comment by psiinon** [comment]
Hopefull all fixed 😁 
I'll have to remember to copy this help next time, I always seem to copy ones which have these issues 😉

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**